lua/conform/formatters/blue.lua

```diff
@@ -0,0 +1,21 @@
+local util = require("conform.util")
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/grantjenks/blue",
+    description = "Blue -- Some folks like black but I prefer blue.",
```

Comment:
Could you make this a bit more useful? The tagline on the repo is `The slightly less uncompromising Python code formatter.` and is more descriptive

Reply:
Sure, done! I believe this was previous tagline on the repo, but creator moved it to README :)